Library/Homebrew/vendor/backports/string.rb

```diff
@@ -0,0 +1,32 @@
+# Taken from https://github.com/marcandre/backports/blob/v3.8.0/lib/backports/2.4.0/string/match.rb
```

Comment:
Can the license and other metadata go in `Library/Homebrew/vendor/README.md` to be consistent with the other vendored stuff?